src/IncludedCommands/Resources/PersistentBans.luau

```diff
@@ -55,69 +56,71 @@ function PersistentBans.Initialize(self: PersistentBans): ()
     if self.InitializationStarted then return end
     self.InitializationStarted = true
 
-    --Get the DataStore.
+    if RunService:IsStudio() then
+        print("NexusAdmin PersistentBans disabled in studio.")
```

Comment:
I should have commented this before - I would prefer this print statement to be removed. This will always pollute the logs on top of what `NexusDataStore` pollutes on every play session. This message will be displayed either way when trying to use the commands.